pkgs/development/libraries/qt-6/modules/qtdeclarative.nix

```diff
@@ -1,9 +1,15 @@
-{ qtModule, lib, python3, qtbase, qtsvg }:
+{ qtModule
+, qtbase
+, libglvnd, libxkbcommon, vulkan-headers # TODO should be inherited from qtbase
+, qtshadertools
+, openssl
+, python3
+}:
 
 qtModule {
   pname = "qtdeclarative";
```

Comment:
note: build is failing
build time is 2.5 hours on dual core
WIP splitBuildInstall